docs/faq.md

```diff
@@ -427,3 +427,35 @@ You need to change kGREYConfigKeyScreenshotDirLocation in GREYConfiguration to c
 [[GREYConfiguration sharedInstance] setValue:@"screenshot_dir_path"
                                 forConfigKey:kGREYConfigKeyScreenshotDirLocation];
 ```
+
+**How do I run tests against a binary app?**
```

Comment:
nit:
binary app => precompiled app?
since you're using precompiled app below as well.